runtime/js/40_http.js

```diff
@@ -172,14 +185,21 @@
                 value,
               );
             } catch (error) {
+              const connError = connErrors.get(httpConn);
+              if (error instanceof errors.BadResource && connError != null) {
+                // deno-lint-ignore no-ex-assign
+                error = new connError.constructor(connError.message);
+              }
               await reader.cancel(error);
               throw error;
             }
           }
         } finally {
           // Once all chunks are sent, and the request body is closed, we can
           // close the response body.
-          await Deno.core.opAsync("op_http_response_close", responseBodyRid);
+          try {
+            await Deno.core.opAsync("op_http_response_close", responseBodyRid);
+          } catch { /* pass */ }
```

Comment:
The ignored catch could perhaps use a more informative comment on why it can be passed.

Reply:
It's because it may already be closed. We have this in other places, a comment is there at all because of the linter.